WordPress/WordPressUITests/Flows/LoginFlow.swift

```diff
@@ -45,14 +68,25 @@ class LoginFlow {
         }
 
         XCTContext.runActivity(named: "Return to app prologue screen if needed") { (activity) in
-            if !WelcomeScreen.isLoaded() {
-                while LoginPasswordScreen.isLoaded() || LoginEmailScreen.isLoaded() || LinkOrPasswordScreen.isLoaded() || LoginSiteAddressScreen.isLoaded() || LoginUsernamePasswordScreen.isLoaded() || LoginCheckMagicLinkScreen.isLoaded() {
-                    if LoginEmailScreen.isLoaded() && LoginEmailScreen.isEmailEntered() {
-                        LoginEmailScreen().emailTextField.clearTextIfNeeded()
+            if !PrologueScreen.isLoaded() {
+                while PasswordScreen.isLoaded() || GetStartedScreen.isLoaded() || LinkOrPasswordScreen.isLoaded() || LoginSiteAddressScreen.isLoaded() || LoginUsernamePasswordScreen.isLoaded() || LoginCheckMagicLinkScreen.isLoaded() {
+                    if GetStartedScreen.isLoaded() && GetStartedScreen.isEmailEntered() {
+                        GetStartedScreen().emailTextField.clearTextIfNeeded()
                     }
                     navBackButton.tap()
                 }
             }
+
```

Comment:
This test currently fails on XCode 12 within develop. It will be uncommented when it is fixed.